.travis.yml

```diff
@@ -1,2 +1,6 @@
+notifications:
+  email: false
+
 rvm:
-  - 2.1.0
+  - 2.1.8
+  - 2.2.4
```

Comment:
Why do we need 2.1.8?  Add ruby 2.3.x?  http://rubies.travis-ci.org/

The prod box seems to be running 2.1.2, is there a plan to upgrade it to 2.1.8?  Why not 2.2.4?